src/System.Management.Automation/engine/Modules/TestModuleManifestCommand.cs

```diff
@@ -1,382 +1,382 @@
-/********************************************************************++
-Copyright (c) Microsoft Corporation.  All rights reserved.
---********************************************************************/
-
```

Comment:
In gitk the change shows ok. not sure if it is because I am using vs 2017. I am updating it again using notebook to give it a try.
![image](https://user-images.githubusercontent.com/16246343/29187084-4d19b514-7dc3-11e7-87dd-b486dd3cc746.png)